src/components/WallsConfig/index.js

```diff
@@ -122,28 +123,22 @@ ConfigWalls.propTypes = {
 };
 
 const mapDispatchToProps = dispatch => ({
-  handleBackButton: () => dispatch({ type: 'HIDE_WINDOWS_PROPS' }),
+  handleBackButton: () => dispatch(hideMainFormLayout()),
```

Comment:
Probably I'm wrong but I think would be better put this logic in the component as methods in order to just write something like 
```export default connect(
  mapStateToProps,
  {hideMainFormLayout, moreActions, etc}
)(MyComponent)
```

Reply:
It could be done in future PRs.